<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <link rel="stylesheet" href="styles.css">
    <link
      rel="stylesheet"
      href="https://use.fontawesome.com/releases/v5.8.2/css/all.css"
    />
    <title>Document</title>
</head>
<body>
  <header>
    <nav class="navbar">
      <h1 class="logo">SJ</h1>
      <ul>
          <li><a href="index.html" class="nav-link">Home</a></li>
          <li><a href="#" class="nav-link">Background Knowledge</a>
              <div class="dropMenu" id="dropMenuBK">
                  <ul>
                      <li><a class="nav-link" href="genetics.html">Genetics</a></li>
                      <li><a class="nav-link" href="statistics.html">Statistics</a></li>
                  </ul>
              </div>
          </li>    
          <li><a href="#" class="nav-link main-link">Research Papers</a> 
              <div class="dropMenu" id="dropMenuResearch">
                  <ul>
                      <li><a class="nav-link" href="research-paper-1.html">Research#1</a></li>
                      <li><a class="nav-link" href="research-paper-2.html">Research#2</a></li>
                      <li><a class="nav-link" href="research-paper-3.html">Research#3</a></li>
                  </ul>
              </div>
          </li> 
      </ul>
      </nav>
  </header>
  <h1>Resetting the bar: Statistical significance in whole-genome
    sequencing-based association studies of global populations</h1>
    <section>
      Minor allele frequency: the amount of times a minor allele is found in a given population over time.
Snps Array: A container of holds snp positions mixed with DNA to determine the alleles of the snps on that array.
Genomic Region: A place where chromatin states can be detected.
Principal component analysis: A collection of points that summarizes large data tables.
Exome: sequence of exons in a sequence.
Singular nucleotide variant (SNV): 
Haplotypes: group of alleles inherited from a single parent.
recombination map: a linkage map based on frequencies 

    </section>
    <section>
      Is significance threshold of P = 5 × 10−8 a good standard?
In GWAS we only observe SNP. In SBAS we 
Quantitle quantile plots have 
PLINK is a package.
Used PCA and QQ plots to see the simulation data.
We made the p value for Africans because they had more genomic variations. They halved the sample size of africans compared to non-africans.
HAPGEN cant’ take into account of newly observed variation.
Mutations that do not manifest phenotypically doesn’t really matter much.
Testing burden: Computational side/statistical side. More tests we do there is more likely a chance to get things wrong.
Downsizing the samples to the same size helps reduce criticism of the results. 
    </section>
  
<section>Methods</section>
<section>Conclusion</section>
  <footer>
    <div class="contact-section" id="author-contact-information">
    <label > Author Contact Information</label>
       <div class="social-icons">
        <a href="https://github.com/anguyen6262"><i class="fab fa-github"></i></a>
        <a href="https://www.linkedin.com/in/andrew-nguyen-6b72b0216/"><i class="fab fa-linkedin-in"></i></a>
        <a href="https://www.facebook.com/profile.php?id=100079858971896"><i class="fab fa-facebook"></i></a>
        <a href="https://www.instagram.com/andrew.d.ng/"><i class="fab fa-instagram"></i></a>
    </div>
    </div>
</footer>
</body>
</html>